### DISCUSSION TITLE
DB_QUERY_LIMIT_DEFAULT doesnt work with docker run

### DISCUSSION THREAD

**Discussion by jckytf** [nocodb/nocodb#8998] [General]
I'm trying nocodb out but i want to increase the rows displayed, i tried to apply DB_QUERY_LIMIT_DEFAULT to 100 but UI still only shows 25 rows per page. Here's my docker run statement:

docker run -d --name nocodb \
-v "$(pwd)/docker/nocodb/data/"/nocodb:/usr/app/data/ \
-p 8080:8080 \
-e DB_QUERY_LIMIT_DEFAULT=100 \
nocodb/nocodb:latest

I also tried a version with -e DB_QUERY_LIMIT_DEFAULT="100"

In docker inspect, they both produces:
	"Config": {
		"Hostname": "311718f7d49d",
		"Domainname": "",
		"User": "",
		"AttachStdin": false,
		"AttachStdout": false,
		"AttachStderr": false,
		"ExposedPorts": {
			"8080/tcp": {}
		},
		"Tty": false,
		"OpenStdin": false,
		"StdinOnce": false,
		"Env": [
			"DB_QUERY_LIMIT_DEFAULT=100",
			"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
			"LITESTREAM_S3_SKIP_VERIFY=false",
			"LITESTREAM_RETENTION=1440h",
			"LITESTREAM_RETENTION_CHECK_INTERVAL=72h",
			"LITESTREAM_SNAPSHOT_INTERVAL=24h",
			"LITESTREAM_SYNC_INTERVAL=60s",
			"NC_DOCKER=0.6",
			"NC_TOOL_DIR=/usr/app/data/",
			"NODE_ENV=production",
			"PORT=8080"
		],

Any help would be appreciated!
Thanks.

**Comment by da-avidid** [comment]
bump.

We have also added the ENV VARS for both "DB_QUERY_LIMIT_MAX=1000 DB_QUERY_LIMIT_DEFAULT=700" and do not see changes reflected in the UI, though the ENV VARS are present in the container.

**Reply by afw384** [reply]
I can confirm this at my installation. DB_QUERY_LIMIT_DEFAULT=100 DB_QUERY_LIMIT_MAX=200 does still set the paging to 25.
